jar-infer/src/main/java/com/uber/nullaway/jarinfer/DefinitelyDerefedParamsDriver.java

```diff
@@ -151,12 +169,13 @@ private static void writeJarModel(
       IClassHierarchy cha, HashMap<IMethod, Set<Integer>> map_mtd_result) {
     try {
       File aStubXFile = new File(aStubXPath);
+      aStubXFile.getParentFile().mkdirs();
       aStubXFile.createNewFile();
       DataOutputStream out = new DataOutputStream(new FileOutputStream(aStubXFile, false));
       Map<String, String> importedAnnotations =
           new HashMap<String, String>() {
             {
-              put("@Nonnull", "javax.annotation.Nonnull");
+              put("Nonnull", "javax.annotation.Nonnull");
```

Comment:
This is fine, but keep in mind there is a small mismatch here where NullAway mostly understands `Nullable`, and assumes no annotation means `Nonnull`, but these models are the reverse. It doesn't really matter, since we will parse them with a separate flow than the one that ingests the code, but maybe for consistency is worth reversing these here?